docs/apache-airflow/macros-ref.rst

```diff
@@ -83,6 +78,27 @@ Variable                                Description
                                         the CLI's test subcommand
 =====================================   ====================================
 
+The following variables are deprecated. They are kept for backward compatibility,
+but you should convert existing code to use other variables instead.
+
+=====================================   ====================================
+Deprecated Variable                     Description
+=====================================   ====================================
+``{{ execution_date }}``                the execution date (logical date), same as ``logical_date``
+``{{ prev_ds }}``                       the previous execution date as ``YYYY-MM-DD``
+                                        if ``{{ ds }}`` is ``2018-01-08`` and ``schedule_interval`` is ``@weekly``,
+                                        ``{{ prev_ds }}`` will be ``2018-01-01``
+``{{ prev_ds_nodash }}``                the previous execution date as ``YYYYMMDD`` if exists, else ``None``
+``{{ yesterday_ds }}``                  the day before the execution date as ``YYYY-MM-DD``
+``{{ yesterday_ds_nodash }}``           the day before the execution date as ``YYYYMMDD``
+``{{ tomorrow_ds }}``                   the day after the execution date as ``YYYY-MM-DD``
+``{{ tomorrow_ds_nodash }}``            the day after the execution date as ``YYYYMMDD``
+``{{ prev_execution_date }}``           the previous execution date (if available) (`pendulum.Pendulum`_)
+``{{ prev_execution_date_success }}``   execution date from prior successful dag run,
+                                        same as ``prev_logical_date_success``
+``{{ next_execution_date }}``           the next execution date, same as ``next_logical_date``
```

Comment:
```suggestion
``{{ next_execution_date }}``           the next execution date, same as ``data_interval_env``
```

(Yes, it would be the same, but I think we should point people to use the explicit data interval properties)